js/main.js

```diff
@@ -10,9 +10,58 @@ var $artButton = document.querySelector('#next');
 var $backButton = document.querySelector('#previous');
 var $artList = document.querySelector('.art-container-list');
 var $artAddDivRow = document.querySelector('.art-adding-page-row');
+var $artViewPage = document.querySelector('.view-art-page');
+var $displayColumn = document.querySelector('.display-column');
 
+var xhr = new XMLHttpRequest();
+var xhrTwo = new XMLHttpRequest();
```

Comment:
Instead of making `xhr` variables globally, you should make it inside of the function making that request. XHR objects are not meant to be reused.